Bottom: 017707f6d893ed71bdffd64f8034ec0a23e7cad4
Top:    6639fa3faa36a30897a12f5c9c5948162a0f0ce9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 17:32:57 +0000

Refresh of fix-broken-undo-handling-for-r

---

diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index b2952d8..649269e 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -136,7 +136,7 @@ nbcb_select_common (LibraryMenuTypePtr net, int pos, int select_flag)
   int i;
 
   InitConnectionLookup ();
-  ResetConnections (false);
+  ResetConnections (false, false);
   SaveUndoSerialNumber ();
 
   for (i = net->EntryN, entry = net->Entry; i; i--, entry++)
@@ -144,7 +144,7 @@ nbcb_select_common (LibraryMenuTypePtr net, int pos, int select_flag)
       RatFindHook (conn.type, conn.ptr1, conn.ptr2, conn.ptr2, true, true);
   RestoreUndoSerialNumber ();
   SelectConnection (select_flag);
-  ResetConncetions (false);
+  ResetConncetions (false, false);
   FreeConnectionLookupMemory ();
   IncrementUndoSerialNumber ();
   Draw ();
